src/prefect/experimental/cli/worker.py

```diff
@@ -0,0 +1,38 @@
+import typer
+import uvicorn
+
+from prefect.cli._types import PrefectTyper
+from prefect.cli.root import app
+from prefect.experimental.workers.base import BaseWorker
+from prefect.utilities.dispatch import lookup_type
+
+worker_app = PrefectTyper(
+    name="worker", help="Commands for starting and interacting with workers."
+)
+app.add_typer(worker_app)
+
+
+@worker_app.command()
+async def start(
+    worker_name: str = typer.Option(..., "--name"),
+    worker_pool_name: str = typer.Option(...),
+    worker_type: str = typer.Option("process", "--type"),
```

Comment:
Can you use `default=`?

Reply:
I think because `typer.Option` has `default` first followed by `*param_decls` in its signature, if we specify custom flags then we can't use `default=` since default can only be used positionally. I'm not sure if there's a way to work around that.